tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/ContextPropagation.cs

```diff
@@ -17,15 +17,12 @@ private static void Inject<TMessageRequest>(SpanContext context, IDictionary mes
             // Consolidate headers into one JSON object with <header_name>:<value>
             var sb = Util.StringBuilderCache.Acquire(Util.StringBuilderCache.MaxBuilderSize);
             sb.Append('{');
-            SpanContextPropagator.Instance.Inject(context, sb, &Setter);
+            SpanContextPropagator.Instance.Inject(context, sb, (StringBuilder carrier, string key, string value) => carrier.AppendFormat("\"{0}\":\"{1}\",", key, value));
```

Comment:
I know this is a revert, but does it help to make these static?
```suggestion
            SpanContextPropagator.Instance.Inject(context, sb, static (StringBuilder carrier, string key, string value) => carrier.AppendFormat("\"{0}\":\"{1}\",", key, value));
```

Reply:
Added in 31aee89. I cannot see any changes in the codegen as expected.

Reply:
> Allow a `static` modifier on lambdas and anonymous methods, which disallows capture of locals or instance state from containing scopes.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/static-anonymous-functions